cmd/world/root/root.go

```diff
@@ -44,6 +52,63 @@ func init() {
 	logger.AddLogFlag(doctorCmd)
 }
 
+func checkLatestVersion() error {
+	url := "https://api.github.com/repos/Argus-Labs/world-cli/releases/latest"
+
+	// Create a request with the context
+	req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, url, nil)
+	if err != nil {
+		return fmt.Errorf("error creating request: %s", err.Error())
+	}
+
+	// Create a new HTTP client and execute the request
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return fmt.Errorf("error fetching the latest release: %s", err.Error())
+	}
+	defer resp.Body.Close()
+
+	// Release structure to hold the data of the latest release
+	type Release struct {
```

Comment:
define struct at the the top of the file

Reply:
updated